modules/jaeger-config-ingester.adoc

```diff
@@ -85,3 +80,28 @@ spec:
       es:
         server-urls: http://elasticsearch:9200
 ----
+
+== Configuring Ingester for autoscaling
+
+The Ingester can be configured to autoscale on demand. You configure autoscaling by setting the `autoscale` parameter to `true` and specifying a value for `.spec.ingester.maxReplicas` along with a reasonable value for the resources that you expect the Ingester's pod to consume. If you do not set a value for `.spec.ingester.maxReplicas` the Operator will set it to `100`.
```

Comment:
I think the last part of the last sentence 

>the Operator will set it to `100`. 

Should be:

>the default is `100`.

I think the operator less anthropomorphic.